package.json

```diff
@@ -45,5 +45,5 @@
   "name": "pcdm-store-updater",
   "preferGlobal": false,
   "private": false,
-  "version": "0.0.1"
+  "version": "1.0.0"
```

Comment:
I'm giving this a proper tagged version so that we can include specific versions of this app inside the discovery-hybrid-indexer.